keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepository.kt

```diff
@@ -41,7 +41,7 @@ interface ArtifactRepository : PeriodicallyCheckedRepository<DeliveryArtifact> {
 
   fun store(artifact: DeliveryArtifact, version: String, status: ArtifactStatus?): Boolean
 
-  fun updateArtifactMetadata(name: String, type: ArtifactType, version: String, status: ArtifactStatus?, artifactMetadata: ArtifactMetadata)
+  fun updateArtifactMetadata(name: String, type: ArtifactType, version: String, status: ArtifactStatus?, artifactMetadata: ArtifactMetadata) : Boolean
```

Comment:
You added the return type but it doesn't look like you're using it anywhere?

Reply:
Will remove it!